Airports/EGLL/Ownership.txt

```diff
@@ -71,18 +71,6 @@ ALTOWNER:Mentoring TWR/APP:LLTM:LLTN:LLT:LLNM:LLN:LLS:LLF:TCSE:TCS:TC:LD:LS:LWS:
 BORDER:LLTWR
 DEPAPT:EGLL
 
-SECTOR:WUAPP:0:2000
```

Comment:
This was in this file because sectors need to be defined in size order. Because the EGWU RMA fits inside the EGLL, it needs to be defined first, hence will need to stay where it is to function correctly

Reply:
This functioned correctly in the tests we conducted - what is the issue that will be caused?

Reply:
I'll give this a test multiple ways to see if this needs to be moved

Reply:
It might work fine from Northolt's perspective (as in it looks as if they have the right ownership), but EuroScope works through the .ese as the sectors are ordered in the file. So if Aircraft A is in the WUAPP sector and the WUAPP sector appears first in the file, then it will match the two together. But if Aircraft A is in the WUAPP sector but also the LLFIN_E sector, and the LLFIN_E is defined first in the .ese, then it will determine it is in LLFIN_E's airspace. This then also affects auto-handoffs.

Reply:
For this reason, the EGMC sectors are in the EGLC ownership file as well, for example: https://github.com/VATSIM-UK/UK-Sector-File/blob/master/Airports/EGLC/Ownership.txt#L43

Reply:
I understand - thanks for that. I'll make some changes but will probably need some further input on how to integrate the VV ownership into the tree.

Reply:
Well the short answer is that I don't think it is possible to make it work in the way that is desired. You cannot have an aircraft within two sectors at any one time; EuroScope matches the first sector it can, and then leaves it at that. If two sectors have slightly different airspace, then the use of GUEST ownership can help to resolve some of the 'gaps', allowing someone to keep an aircraft while it transit's through another sector's airspace, for example. But ultimately, putting VV above our civilian sectors would screw everything over and putting it below wouldn't really have much use except to show the military controller their AoR.

This is the main reason why I had been chatting to Andy about the possibility of using the new generator to produce both a civilian and a military version of the sector file. Each would have a good chunk of the same data, but then SMRs, agreements, ownership etc. that are irrelevant to one or the other could be removed. Hence why it might be a good idea if this PR were split up into parts so that what can be merged now is merged, but some other bits might have to wait until a bigger plan is worked out

Reply:
Showing the AoR and active boundaries to the swanwick sectors is the goal - clearly there's no point having the vast majority of traffic appearing as within the Swanwick sectors, and it should go below any of the civilian sectors to avoid interference. The SIL and predictions are irrelevant to Swanwick really.

The other goal is to make departure/arrival and METAR lists work correctly (automatically) for the airfields covered top down. Currently military controllers have to manually select owned airfields in the list with no indication of when local controllers come online. This required changing and adding local ownership files.

As for a military only sector file, that used to be the philosophy and we've been deliberately moving away from that because of the excessive workload required to keep it updated. Having one produced seperately from the main GitHub  could solve this issue, but it sounds like a lot of work to get it working and not sure what the benefits would be?

Reply:
That's not quite what I was getting at - both would use the same single repository, as they are doing now - but the generator could cherry pick the files relevant to either military or civilian controllers and produce two separate files. The idea being that there is no excessive workload to keep it updated! From my chat with Andy there isn't a lot of work involved, because the generator is already built around defining a config file that determines what's included. 

The reason this is useful though is mainly to do with optimisation: the sector file and its sectors are already a drain on resources for those with slower PCs, and it performs particularly badly during busy events; the file size is also continuing to spiral upwards, which makes for an ever-increasing load time when opening EuroScope on less powerful devices which could be reduced. Splitting the files up would mean that the civilian version does not need to have any unnecessary SMRs or other airfield-related data/ownership, and it would perhaps allow for more sophisticated military ownership that wouldn't interfere with the civilian side, if it's needed.

Reply:
Sounds interesting! I have changed this issue and pushed a new version of this file.